nixos/modules/system/boot/initrd-network.nix

```diff
@@ -45,7 +45,8 @@ in
         is acquired using DHCP.
 
         You should add the module(s) required for your network card to
-        boot.initrd.availableKernelModules. lspci -v -s &lt;ethernet controller&gt;
+        boot.initrd.availableKernelModules.
+        <literal>lspci -v | grep -iA8 'network\|ethernet'</literal>
```

Comment:
I am not used to the doc but shouldn't it be <command> instead ?

Reply:
Be what? :relaxed:

Reply:
github hid the text between < > xD

Reply:
Nice catch, fixed.

Reply:
After a closer look it seems that NixOS docs and descriptions mostly use `<literal>` for complex shell commands:
```
grep -r '<literal>.*|' $(nix eval --raw '(builtins.fetchTarball { url = https://nixos.org/channels/nixos-unstable/nixexprs.tar.xz; })')
# vs
grep -r '<command>.*|' $(nix eval --raw '(builtins.fetchTarball { url = https://nixos.org/channels/nixos-unstable/nixexprs.tar.xz; })')
```
I've reverted the change.